### ISSUE TITLE
Normal model behavior ? or scaling ?

### ISSUE THREAD

**Issue by YacineDeghaies** [DepthAnything/Depth-Anything-V2#106]
Hi,
I've continued training the Depth-Anything-V2-Large model serial on my custom dataset, and the predicted depth maps appear to be close to the ground truth ones. However, they appear to be scaled for some reason, which is evident from the black and white pixels(see annotations on images). I'm unsure if this is normal behavior from the model itself of if there's some scaling happening in the code.

predicted depth map:

![depth_shot_0003_source_0001](https://github.com/user-attachments/assets/58b78307-e406-42e9-a8dd-cf9a8d948357)

ground truth:
![shot_0003_dm_gt_s_0000](https://github.com/user-attachments/assets/0df25dac-9f41-4a2b-8df4-58438b10f190)

**Comment by shilpaullas97** [comment]
> Hi, I've continued training the Depth-Anything-V2-Large model serial on my custom dataset, and the predicted depth maps appear to be close to the ground truth ones. However, they appear to be scaled for some reason, which is evident from the black and white pixels(see annotations on images). I'm unsure if this is normal behavior from the model itself of if there's some scaling happening in the code.
> 
> predicted depth map:
> 
> ![depth_shot_0003_source_0001](https://private-user-images.githubusercontent.com/56239998/351719013-58b78307-e406-42e9-a8dd-cf9a8d948357.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY1NzE2ODQsIm5iZiI6MTcyNjU3MTM4NCwicGF0aCI6Ii81NjIzOTk5OC8zNTE3MTkwMTMtNThiNzgzMDctZTQwNi00MmU5LWE4ZGQtY2Y5YThkOTQ4MzU3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTE3VDExMDk0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgwZjE5OTNjODA0NzdhZWY5ZjQyYzFmODIxMWM4ZGFkN2U5MTFkM2Y4ZmE3MzJkNzFlMzIzNGUyM2Y4OGFkMDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.80cQ0yNJGfFwZPBpRWRDSc_PYbAIGthSpx-seA9kxnc)
> 
> ground truth: ![shot_0003_dm_gt_s_0000](https://private-user-images.githubusercontent.com/56239998/351718359-0df25dac-9f41-4a2b-8df4-58438b10f190.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY1NzE2ODQsIm5iZiI6MTcyNjU3MTM4NCwicGF0aCI6Ii81NjIzOTk5OC8zNTE3MTgzNTktMGRmMjVkYWMtOWY0MS00YTJiLThkZjQtNTg0MzhiMTBmMTkwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTE3VDExMDk0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJlMDU2NmQ2OTI1ZTc1ZjFlYjI3ZDVkZDJkMjcxMmY0OWM3MzM3ZjNmY2FlYTFjYjk3ODhlMTI5YTFjZTliNmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.cJbeKVtQwKoWooIkAvRaaRdmZqGfvTDsQ8TxpTgjITo)

Hi @YacineDeghaies ,

I'm currently trying to train metric depth model on my custom dataset. I was able to run the training code, but my predicted depth maps are worse compared to the groundtruth I have.
Could you please share the steps you followed for training on custom dataset.
